plugins/for_each_line.cc

```diff
@@ -238,17 +238,17 @@ forEachLineStr(const std::string & filename,
 /*****************************************************************************/
 
 void forEachLineBlock(std::istream & stream,
-                      int64_t lineOffset,
```

Comment:
Sure, I can do that.  However, all the other functions in for_each_line have this extra parameter and this function `forEachLineBlock` was also defining a `maxLines` variable internally (see below).